core/src/main/java/org/codice/imaging/nitf/core/impl/HeapStrategyConfiguration.java

```diff
@@ -14,7 +14,7 @@
  */
 package org.codice.imaging.nitf.core.impl;
 
-import java.util.function.Predicate;
+import java.util.function.LongPredicate;
```

Comment:
Who knew Java had a LongPredicate? Not me.

Reply:
👏 Sonar has our back 👏